azurecompute/src/main/java/org/jclouds/azurecompute/options/AzureComputeTemplateOptions.java

```diff
@@ -89,6 +89,14 @@ public void copyTo(final TemplateOptions to) {
          if (storageAccountType.isPresent()) {
             eTo.storageAccountType(storageAccountType.get());
          }
+
+         final LoginCredentials credentials = new LoginCredentials.Builder().
+                 user(loginUser).
+                 password(loginPassword == null || !loginPassword.isPresent() ? null : loginPassword.get()).
+                 privateKey(loginPrivateKey == null || !loginPrivateKey.isPresent() ? null : loginPrivateKey.get()).
+                 authenticateSudo(authenticateSudo == null ? false : authenticateSudo).
+                 build();
+         eTo.overrideLoginCredentials(credentials);
```

Comment:
Without this, I get failure in [BaseTemplateBuilderLiveTest#testFromTemplate](https://github.com/jclouds/jclouds/blob/master/compute/src/test/java/org/jclouds/compute/internal/BaseTemplateBuilderLiveTest.java#L65) because `loginPasswordPresent=true, loginPrivateKeyPresent=true` is not correctly copied.